### PR TITLE
PMM-9951 Adding PGSM+PMM Integration tests

### PR DESCRIPTION
**Pull request by puneet0191**
I had to close the previous PR due to some weird commit conflict

### CODE REVIEW

**Review by capri1989 — APPROVED** [review]
LGTM

File: .github/workflows/postgresql-13-pmm.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          repository: 'Percona-Lab/qa-integration'
+          ref: 'main'
+
+      # print branch name
+      - name: Get branch name
+        run: echo 'The branch name is' ${{ github.head_ref }}
+      
+      - name: Get current path
+        run: pwd
+
```
Comment (EngineeredVirus) [inline .github/workflows/postgresql-13-pmm.yaml:22]: ISTM that these lines are only for version 13. Why are these needed here?

Comment (puneet0191) [reply .github/workflows/postgresql-13-pmm.yaml:22]: I think I forgot to remove them, it was added for debugging purpose as I recall

**Review by EngineeredVirus — APPROVED** [review]
LGTM
